lib/string-checker.js

```diff
@@ -130,7 +130,12 @@ StringChecker.prototype = {
 
         this._configuredRules.forEach(function(rule) {
             errors.setCurrentRule(rule.getOptionName());
-            rule.check(file, errors);
+
+            try {
```

Comment:
Idk why but `time mocha` execution time like 5% lower in branch than in master. Called several times... results the same.

`issues/1271`:
```
  3497 passing (17s)

real    0m18.497s
user    0m16.493s
sys     0m0.408s
```

`master`:
```
  3490 passing (20s)

real    0m21.552s
user    0m19.225s
sys     0m0.460s
```